vertical-pod-autoscaler/hack/generate-crd-yaml.sh

```diff
@@ -53,4 +53,5 @@ resources:
 commonAnnotations:
   "api-approved.kubernetes.io": "https://github.com/kubernetes/kubernetes/pull/63797"
 EOF
-kubectl kustomize . > ${OUTPUT}
+echo --- > ${OUTPUT}
+kubectl kustomize . >> ${OUTPUT}
```

Comment:
Why those changes?

Reply:
Apparently kustomize doesn't put the `---` at the top of the first YAML document when there are more than one. Most of the other YAML files in `deploy` have the leading `---` so I thought to add it to make them more consistent, although it's optional on the first document.  I'm happy to remove it if it seems hacky.